Apps/Common/SIMCoupled.h

```diff
@@ -53,7 +53,13 @@ template<class T1, class T2> class SIMCoupled
   }
 
   //! \brief Computes the solution for the current time step.
-  virtual bool solveStep(TimeStep& tp, bool firstS1 = true)
+  virtual bool solveStep(TimeStep& tp)
+  {
+    return this->solveStep(tp, true);
+  }
+
+  //! \brief Computes the solution for the current time step.
+  virtual bool solveStep(TimeStep& tp, bool firstS1)
```

Comment:
The point is that solveStep(tp), could either mean the first method or the second method with its default param. When we then add the using statement to quell warnings, we get two overloads in children.